openy_gated_content.module

```diff
@@ -129,3 +130,49 @@ function _openy_gated_content_form_eventseries_live_stream_submit(&$form, FormSt
 function openy_gated_content_page_attachments(array &$attachments) {
   $attachments['#attached']['library'][] = 'openy_gated_content/styles';
 }
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function openy_gated_content_form_user_login_form_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $form['#validate'][] = 'openy_gated_content_user_login_validation';
+}
+
+/**
+ * Implements hook_form_FORM_ID_alter().
+ */
+function openy_gated_content_form_user_pass_alter(&$form, FormStateInterface $form_state, $form_id) {
+  $form['#validate'][] = 'openy_gated_content_user_pass_validation';
+}
+
+/**
+ * Checks if user has related to Virtual Y role.
+ */
+function openy_gated_content_user_login_validation(array &$form, FormStateInterface $form_state) {
+  $account = \Drupal::entityTypeManager()->getStorage('user')->load($form_state->get('uid'));
+  if ($account) {
+    $roles = $account->getRoles();
+    foreach ($roles as $role) {
+      if (strpos($role, 'virtual_y')) {
```

Comment:
```suggestion
      if (strpos($role, 'virtual_y_')) {
```

Reply:
@jediandrey @hamrant we have Virtual Y editor role that will be blocked without my fix above ^^
![image](https://user-images.githubusercontent.com/3023950/93066581-d6507580-f682-11ea-8b9c-10fb4cb22270.png)